plugins/foreman_openscap/0.5/index.md

```diff
@@ -17,8 +17,8 @@ OpenSCAP reports (aka ARF reports) will help you find vulnerabilities on your ho
 Foreman OpenSCAP plugin is made of 5 components:
 
 * [foreman_openscap](https://github.com/theforeman/foreman_openscap) - Foreman plugin which creates and persists SCAP content, compliance policy and ARF report objects.
-* [smart_proxy_openscap](https://github.com/theforeman/smart_proxy_openscap) - Smart-Proxy plugin which validates SCAP content from Foreman, distributes SCAP content to clients and posts ARF reports from client hosts to Foreman
-* [foreman_scap_client](https://github.com/theforeman/foreman_scap_client) - A client script which runs OpenSCAP scan and uploads the scan report to the Smart-Proxy
+* [smart\_proxy\_openscap](https://github.com/theforeman/smart_proxy_openscap) - Smart-Proxy plugin which validates SCAP content from Foreman, distributes SCAP content to clients and posts ARF reports from client hosts to Foreman
```

Comment:
I don't think escaping underscores is needed here, the [manual](http://www.theforeman.org/plugins/foreman_openscap/0.5/index.html#1.ForemanOpenSCAPmanual) looks fine